data/.gitignore

```diff
@@ -0,0 +1,64 @@
+/captures
```

Comment:
this is not necessary as you have a project gitIgnore.

Reply:
done